### DISCUSSION TITLE
How can I use Explorer on React project?

### DISCUSSION THREAD

**Discussion by ghost** [Smithsonian/dpo-voyager#103] [General]
How can I use Explorer on React project? Examples contain only how to embed Explorer component on a regular web page.

**Comment by gjcope** [comment]
Hi @mkoske ! I haven't had time to put together a React example but I believe one of my colleagues (@shintung00) has experience with it, so I will ask for his input. Stay tuned.

**Comment by shintung00** [comment]
Ask and ye shall receive @mkoske.
Here's the [CodeSandbox demo](https://codesandbox.io/s/br29b) of creating an Explorer React component.

Feel free to reach out if you have any more questions!

**Reply by ghost** [reply]
Thanks for the example. I'll look into that better when I have time, but seems quite simple.
